workflows/Lookup_Vuln_From_Slack/workflow.spec.yaml

```
@@ -12,7 +12,7 @@ hub_tags:
   keywords: [remediation_management, vulnerability_management, chat, chatops]
   features: []
 resources:
-  source_url: https://github.com/rapid7/insightconnect-workflows/tree/master/workflows/Lookup_Vuln_From_Slack
+  source_url: https://github.com/rapid7/insightconnect-workflows/tree/master/workflows/Lookup_Vulnerability_From_Slack
```

This needs to be reverted
```suggestion
  source_url: https://github.com/rapid7/insightconnect-workflows/tree/master/workflows/Lookup_Vuln_From_Slack
```